src/commands/lambda/constants.ts

```diff
@@ -115,5 +115,6 @@ export const MAX_LAMBDA_STATE_CHECK_ATTEMPTS = 3
 export const EXTRA_TAGS_REG_EXP: RegExp = /^(([a-zA-Z]+)\w+:[\w\-/\.]+)+((\,)([a-zA-Z]+)\w+:[\w\-/\.]+)*$/g
 export const AWS_ACCESS_KEY_ID_REG_EXP: RegExp = /(?<![A-Z0-9])[A-Z0-9]{20}(?![A-Z0-9])/g
 export const AWS_SECRET_ACCESS_KEY_REG_EXP: RegExp = /(?<![A-Za-z0-9/+=])[A-Za-z0-9/+=]{40}(?![A-Za-z0-9/+=])/g
+export const AWS_SECRET_ARN_REG_EXP: RegExp = /arn:aws:secretsmanager:[\w-]+:\d{12}:secret:.+/
```

Comment:
Should we be testing for KMS too?

Reply:
Good point, I just started a slack thread